src/app/jumbotron/jumbotron.component.scss

```diff
@@ -0,0 +1,131 @@
+$breakpoint-mobile-small: 325px;
+$breakpoint-mobile: 425px;
+$breakpoint-tablet: 768px;
+$breakpoint-laptop:1024px;
+$Picton-Blue:#2AB1EA;
+$White:#FFFFFF;
+$Black : #000000;
+$Haiti : #0B132B;
+.custom-jumbo{
+    height: 433px;
+    width: 100%;
+}
+.img-div
+{
+    margin-top: 30px;
+}
+.home-img {
+	height: 315px;
+	width: 501px;
+}
+.align
+{
+    text-align: center;
+}
+.empowering-employabi {
+	height: 42px;
+	opacity: 0.8;
+	color: $Black ;
+	font-size: 36px;
+	font-weight: bold;
+    line-height: 42px;
+    display: initial;
+    font-family: Raleway;
+}
+.brainteaser{
+	height: 19px;
+	width: 530px;
+	color:$Haiti; 
+	font-family: Raleway;
+	font-size: 16px;
+	line-height: 19px;
+}
+.jumbotron p {
```

Comment:
use nested scss

Reply:
and do not target elements. Use css class names only